fv3core/stencils/tracer_2d_1l.py

```diff
@@ -170,7 +170,14 @@ def __init__(self, comm: fv3gfs.util.CubedSphereCommunicator, namelist):
             domain=self.grid.domain_shape_compute(),
             externals=local_axis_offsets,
         )
-        self.finite_volume_transport = FiniteVolumeTransport(namelist, namelist.hord_tr)
+        self.finite_volume_transport = FiniteVolumeTransport(
+            grid=self.grid.grid_indexing,
```

Comment:
Why are you setting this differently here compared to the call in d_sw? e.g. grid=GridIndexing.from_legacy_grid(self.grid), vs. self.grid.grid_indexing. Either way is fine, but is less confusing if we use the same convention

Reply:
Only because I hadn't written the `grid_indexing` method when I wrote the change for d_sw. I'll refactor it there.